src/ast/test/tests.cmake

```diff
@@ -0,0 +1,44 @@
+if(NOT EXISTS ${PARSER})
+  message(FATAL_ERROR " To run tests you must build the install target." ${PARSER})
+endif()
+
+find_package(Python3 COMPONENTS Interpreter)
+
+if (NOT Python3_FOUND)
+  message(WARNING " No python interpreter - Test Suite not compiled")
+  return()
+endif()
+
+find_program(FILECHECK NAMES OutputCheck)
+
+if(NOT FILECHECK)
+  message(WARNING " Could not find OutputCheck - Test Suite not compiled")
+  message(WARNING "   Run: pip install OutputCheck")
+  return()
+endif()
+
+file(GLOB INPUT_FILES ${TEST_DIR}/*.verona)
```

Comment:
we may want to have multiple sub-dirs for different kinds of tests, can CMake GLOB recurse?

Reply:
It can if you use `GLOB_RECURSE`